### ISSUE TITLE
[Grid] accessibility announcements issue using JAWS

### ISSUE THREAD

**Issue by svetq** [telerik/kendo-angular#3529]
**Describe the bug**
Using a sortable and navigable Grid causes some accessibility announcement issues when using JAWS. It appears that the implementation should be reviewed and updated.

linking a related issue: https://github.com/telerik/kendo-angular/issues/1711#issuecomment-416222924

**To Reproduce**
Please see the following example:
https://stackblitz.com/edit/angular-kvvtfn?file=app/app.component.ts

1. Run JAWS
2. Set focus to first column header
3. Press Enter key. JAWS: "enter key sorted ascending" - correct
4. Press right arrow once and press Enter key. JAWS: "enter not sorted sorted ascending" - wrong

**Comment by tsvetomir** [comment]
Looks like we need to remove the sorting announcement introduced in https://github.com/telerik/kendo-angular/issues/1711. At the time JAWS and NVDA did not announce changes in the `aria-sort` value, but this seems to be fixed now.
